resources/voting-stats.js

```diff
@@ -0,0 +1,112 @@
+let voteChart = null;
+let data = [];
+let min_possible_vote = 1;
+let max_possible_vote = 50;
+
+function set_vote_graph_default_data(default_data, min_allowed_vote, max_allowed_vote) {
+    data = default_data;
+    min_possible_vote = min_allowed_vote;
+    max_possible_vote = max_allowed_vote;
+}
+
+function reload_vote_graph() {
+    if (voteChart !== null) voteChart.destroy();
+
+    if (data.length === 0) {
+        $('#id_no_votes_footer').prop('hidden', false);
```

Comment:
Liines 16-19 looks duplicated with 21-24 except the booleans are reversed. Perhaps we should declare a variable (i.e. `let no_data = (data.length === 0);` so that it's more clear which variables are false/true?

Reply:
I don't think it makes it any better. 
```
$('#id_no_votes_footer').prop('hidden', no_data);
$('#id_vote_chart').prop('hidden', !no_data);        
$('#id_has_votes_footer').prop('hidden', no_data);
$('#id_no_votes_error').prop('hidden', !no_data);
```
It doesn't look any better?

Reply:
I think it makes it better because you save 8 lines, and reduce the if-else to just an if.

Reply:
Honestly, I'm with @magicalsoup : I think declaring a boolean makes it less clear.